app/assets/javascripts/Components/Result/marks_panel.jsx

```diff
@@ -219,7 +233,25 @@ class CheckboxCriterionInput extends React.Component {
   }
 }
 
-class FlexibleCriterionInput extends React.Component {
+CheckboxCriterionInput.propTypes = {
+  id: PropTypes.number.isRequired,
```

Comment:
Yeah so in general grouping/ordering of the props is done pretty inconsistently.

I think the best approach for now is just to order these props alphabetically (and don't include any blank lines).